examples/speech_recognition/model/README.md

```diff
@@ -0,0 +1,20 @@
+# Inter OpenVINO
+
+Intel [OpenVINO](https://docs.openvinotoolkit.org/latest/_inference_engine_samples_speech_sample_README.html) shows how to run the speech sample application, which demonstrates acoustic model inference based on Kaldi* neural networks and speech feature vectors.
```

Comment:
Put this as a reference at last. For example:

### Reference

Please refer to [the speech sample of OpenVINO](https://docs.openvinotoolkit.org/latest/_inference_engine_samples_speech_sample_README.html). It demonstrates acoustic model inference based on Kaldi* neural networks and speech feature vectors.

Reply:
I have update these in the latest commit, PTAL, thanks!

Reply:
README looks good. Thanks!